player/Cargo.toml

```diff
@@ -12,6 +12,11 @@ keywords = ["graphics"]
 license = "MPL-2.0"
 publish = false
 
+[lib]
+
+[[bin]]
```

Comment:
It's my understanding that this isn't necessary at all, cargo will pick up the bin/lib spilt as is.